bucket/steam-library-manager.json

```diff
@@ -1,20 +1,26 @@
 {
-    "version": "1.5.0.13",
+    "version": "1.5.1.7",
     "license": "MIT",
-    "url": "https://github.com/RevoLand/Steam-Library-Manager/releases/download/v1.5.0.13/Steam.Library.Manager.exe",
+    "url": "https://github.com/RevoLand/Steam-Library-Manager/releases/download/v1.5.1.7/Steam.Library.Manager.zip#/dl.7z",
```

Comment:
Rewriting zip to 7zip?? Why?

If it is necessary at comment to manifest about this

Reply:
@Ash258 Good point. I removed the dl.7z as it's unneeded.